src/xtgeo/grid3d/_gridprop_op1.py

```diff
@@ -134,12 +204,6 @@ def operation_polygons(
 
     gl.update_values_from_carray(proxy, cvals, np.float64, delete=True)
 
```

Comment:
The operation proxyv = proxy.values.astype(np.int8) gave several warnings. I cannot recall why a cast was needed, so I just simplify this